src/atom.coffee

```diff
@@ -553,12 +553,16 @@ class Atom extends Model
     @loadConfig()
     @keymaps.loadBundledKeymaps()
     @themes.loadBaseStylesheets()
-    @packages.loadPackages()
+    
+    setTimeout =>
+      @packages.loadPackages()
```

Comment:
If packages are loaded after the editor window is deserialized (`@deserializeEditorWindow()`) then any previously open editors from packages (such as image view, archive view, etc.) won't reopen since their deserializers won't be registered until after deserialization occurs.